### PR TITLE
set up amplitude

### LINKED ISSUE #96: Set up Amplitude Analytics
https://amplitude.com/

### PR DESCRIPTION
**Pull request by OleAndreasER**
Don't freak out about API key.  
image

### CODE REVIEW

File: app/src/main/kotlin/com/platform/openemoji/Activity.kt [hunk 1]
```diff
@@ -3,14 +3,29 @@ package com.platform.openemoji
...
 class Activity : ComponentActivity() {
     override fun onCreate(savedInstanceState: Bundle?) {
         super.onCreate(savedInstanceState)
-
+        val amplitude =
+            Amplitude(
+                Configuration(
+                    // It's safe to expose api key
+                    // https://github.com/amplitude/Amplitude-Javascript/issues/100
+                    apiKey = "efe7c6f94fd4d20f32b6f9b91f66d6be",
```
Comment (michaelbrusegard) [inline app/src/main/kotlin/com/platform/openemoji/Activity.kt:22]: Put a new regenerated api key in gradle.properties and delete this. Also remove it from from amplitude so it cannot be used. api keys should not be hardcoded and they should be manually added by us when we need it

**Review by michaelbrusegard — CHANGES_REQUESTED** [review]

File: app/src/main/kotlin/com/platform/openemoji/Activity.kt (hunk 1 above)
Comment (OleAndreasER) [reply app/src/main/kotlin/com/platform/openemoji/Activity.kt:22]: ...

Comment (OleAndreasER) [reply app/src/main/kotlin/com/platform/openemoji/Activity.kt:22]: image
image
The API key should be hardcoded, and there is no reason to put it anywhere other than here. Can you come up with a reason?

Comment (michaelbrusegard) [reply app/src/main/kotlin/com/platform/openemoji/Activity.kt:22]: OK, I didnt read the comment properly. Doesnt this mean that anyone can give us analytics data without even using the app

Comment (michaelbrusegard) [reply app/src/main/kotlin/com/platform/openemoji/Activity.kt:22]: Im gonna resolve it since it said that it was normal practice to have it exposed

**Review by michaelbrusegard — APPROVED** [review]

File: app/src/main/kotlin/com/platform/openemoji/Activity.kt (hunk 1 above)
Comment (OleAndreasER) [reply app/src/main/kotlin/com/platform/openemoji/Activity.kt:22]: Yes. Even if that would be a problem for Zedge, it's not a problem when we're using my personal api key connected to my Amplitude account. It doesn't seem like that's a problem in general though, if all the data analytic tools tell you to leave the key in the client source code.

**Pull request merged**